### PR TITLE
Fix development group check

### PR DESCRIPTION
**Pull request by mimi1vx**
Add performace optimization when fetching result from openqa
Add test for IncSyncRes

### CODE REVIEW

File: openqabot/openqa.py [hunk 1]
```diff
@@ -12,6 +12,8 @@
 
 logger = logging.getLogger("bot.openqa")
 
+_DEVELOPMENT_PARENT_GROUP_ID = 9
```
Comment (okurz) [inline openqabot/openqa.py:15]: I suggest to move that to __init__.py where we have other configuration settings already

**Review by okurz — CHANGES_REQUESTED** [review]

**Review by okurz — CHANGES_REQUESTED** [review]
@mimi1vx you asked for review but seems you missed the actual suggested changes

**Comment by mimi1vx** [comment]
> @mimi1vx you asked for review but seems you missed the actual suggested changes

oh, .... I forget push

**Review by okurz — DISMISSED** [review]
Code looks good but please still fix the typo in the git commit message

**Review by baierjan — APPROVED** [review]

**Pull request merged**